api/test/helpers/db.coffee

```diff
@@ -22,5 +22,7 @@ fixturize = (collection, data) ->
   data
 
 @empty = (callback) =>
-  cb = _.after @db.collections.length, callback
-  @db[col].drop(cb) for col in @db.collections
+  @db.getCollectionNames (err, names) =>
+    return callback() if names.length is 0
+    cb = _.after names.length, callback
+    @db.collection(col).drop(cb) for col in names
```

Comment:
[Mongojs 2.4](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#300--2016-07-31) only exposes `_getConnection` and `getCollection` -- no longer an array of collections we can iterate over. We can get the collections via the `getCollectionNames` method instead!